src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/MainPageTests.java

```diff
@@ -473,7 +475,8 @@ public void MercuryCuratedMainPageTest_005_CheckElementsVisibilityExceptTrending
   // TODO: add description to test
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)